[2025-07-09T05:49:52.718+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T05:49:52.732+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_airflare_data.extract manual__2025-07-09T05:48:27.734871+00:00 map_index=1 [queued]>
[2025-07-09T05:49:52.738+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_airflare_data.extract manual__2025-07-09T05:48:27.734871+00:00 map_index=1 [queued]>
[2025-07-09T05:49:52.738+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2025-07-09T05:49:52.747+0000] {taskinstance.py:2330} INFO - Executing <Mapped(_PythonDecoratedOperator): extract> on 2025-07-09 05:48:27.734871+00:00
[2025-07-09T05:49:52.756+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4470) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-09T05:49:52.757+0000] {standard_task_runner.py:63} INFO - Started process 4474 to run task
[2025-07-09T05:49:52.757+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_airflare_data', 'extract', 'manual__2025-07-09T05:48:27.734871+00:00', '--job-id', '333', '--raw', '--subdir', 'DAGS_FOLDER/Untitled-1.py', '--cfg-path', '/tmp/tmpcqsux6xf', '--map-index', '1']
[2025-07-09T05:49:52.758+0000] {standard_task_runner.py:91} INFO - Job 333: Subtask extract
[2025-07-09T05:49:52.803+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_airflare_data.extract manual__2025-07-09T05:48:27.734871+00:00 map_index=1 [running]> on host e6a32b72ede0
[2025-07-09T05:49:52.877+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_airflare_data' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T05:48:27.734871+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T05:48:27.734871+00:00'
[2025-07-09T05:49:52.878+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T05:49:52.879+0000] {Untitled-1.py:111} INFO - Начало извлечения данных для направления HKT-IKT
[2025-07-09T05:49:53.631+0000] {Untitled-1.py:26} INFO - Создание подключения к S3
[2025-07-09T05:49:53.632+0000] {Untitled-1.py:31} INFO - Подключение установлено
[2025-07-09T05:49:53.642+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T05:49:53.643+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T05:49:54.007+0000] {Untitled-1.py:37} INFO - Загрузка данных в S3
[2025-07-09T05:49:54.183+0000] {Untitled-1.py:48} INFO - Загрузка завершена успешно
[2025-07-09T05:49:54.184+0000] {Untitled-1.py:124} INFO - Данные успешно сохранены во временное хранилище
[2025-07-09T05:49:54.198+0000] {Untitled-1.py:129} INFO - Данные успешно извлечены и переданы в Xcom
[2025-07-09T05:49:54.200+0000] {python.py:237} INFO - Done. Returned value was: {'bucket': 'airflare', 'key': 'temp/response_HKT_IKT.json'}
[2025-07-09T05:49:54.201+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T05:49:54.218+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_airflare_data, task_id=extract, run_id=manual__2025-07-09T05:48:27.734871+00:00, map_index=1, execution_date=20250709T054827, start_date=20250709T054952, end_date=20250709T054954
[2025-07-09T05:49:54.295+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T05:49:54.379+0000] {taskinstance.py:3503} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2025-07-09T05:49:54.381+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
